roadmap/implementers-guide/src/node/availability/availability-recovery.md

```diff
@@ -89,39 +86,32 @@ On `Conclude`, shut down the subsystem.
 #### `AvailabilityRecoveryMessage::RecoverAvailableData(receipt, session, Option<backing_group_index>, response)`
 
 1. Check the `availability_lru` for the candidate and return the data if so.
-1. Check if there is already an interaction handle for the request. If so, add the response handle to it.
-1. Otherwise, load the session info for the given session under the state of `live_block_hash`, and initiate an interaction with *`launch_interaction`*. Add an interaction handle to the state and add the response channel to it.
+1. Check if there is already an recovery handle for the request. If so, add the response handle to it.
+1. Otherwise, load the session info for the given session under the state of `live_block_hash`, and initiate a recovery task with *`launch_recovery_task`*. Add a recovery handle to the state and add the response channel to it.
 1. If the session info is not available, return `RecoveryError::Unavailable` on the response channel.
 
-### From-interaction logic
+### Recovery logic
 
-#### `FromInteraction::Concluded`
-
-1. Load the entry from the `interactions` map. It should always exist, if not for logic errors. Send the result to each member of `awaiting`.
-1. Add the entry to the `availability_lru`.
```

Comment:
This logic is still there when `remote` is ready.

Reply:
The whole part regarding how this is handled is rather abstract in the guide compared to the actual impl. Either this needs a revamp (the guide) or we agree that details like these don't necessarily belong into the impl guide, since we are entering a more stable dev cycle.